doc/Overview/Mvux/ListFeeds.md

```diff
@@ -0,0 +1,78 @@
+---
+uid: Overview.Mvux.ListFeeds
+---
+
+# What are List-Feeds?
+
+The `IListFeed<T>` is like a [Feed](xref:Overview.Mvux.Feeds) which is stateless and keeps no track of changes, but is specialized for handling collections.
+
+In Feeds, each data request from the service, returns one single item, whereas in List-Feed each request returns a collection of items.  
+Its distinctive feature is the ability to use its [Operators](xref:Overview.Mvux.ListFeeds#operators) directly on its items, rather than on the entire collection.
+
+Another notable characteristic of List-Feed is how it handles the absence of data. When an empty collection is returned by the service, it's treated as an Empty message. The returned data axis Option will be `None`, even though the result was not `null`.  
+The reasoning behind that is so that is that when we display a control of data items (for instance a `ListView`, and it has no items, there's no reason in displaying the `FeedView`'s `ValueTemplate` with an empty `ListView` but the "No data records" `NoneTemplate` which makes much more sense. Hence, a `null` result or an empty collection are both regarded as `None`.
```

Comment:
```suggestion
This is because when a control of data items is displayed with an empty collection (for instance a `ListView`), there is no reason to display the `FeedView`'s `ValueTemplate` with an empty `ListView`. The "No data records" `NoneTemplate` makes much more sense in this case. For that reason, both a `null` result and empty collection are regarded as `None`.
```